src/components/Delegation/StakingDashboard.js

```diff
@@ -717,10 +717,10 @@ class StakingDashboard extends React.Component<Props, State> {
   }
 }
 
-type ExternalProps = {|
+type ExternalProps = {
```

Comment:
why some props need top be unsealed?

Reply:
react-navigation passes in `Route` and `Navigation`
it is a flow error to pass in a prop that is not specified in a sealed prop type

Reply:
mhm then probably better to add `Route` in the types object and keep the object sealed

Reply:
the only place in the flow docs i see sealed props for react components is defaultProps for class components
all the other instances in the flow docs use unsealed props

what are the benefits of sealed props?

Reply:
and you would have to pass those props in for storybook, even though the component doesn't use them

Reply:
Using exact objects is recommended as a general rule. It may prevent some bugs. Also, I believe they wanted to use exact objects as the default behaviour in the future.

for storybook there's already a decorator that passes on mocks of `navigation` and `route` to every component

Reply:
what bugs could it prevent?

Reply:
For instance if during development you instantiate a component with an extra prop, and you haven't added the code that handles that prop yet, you won't see any error if props are not sealed.